src/components/cards/index.ts

```diff
@@ -0,0 +1,7 @@
+export { ActionCard } from './ActionCard';
+export { DPICard } from './DPICard';
+export { ENSSearchCard } from './ENSSearchCard';
+export { ENSCreateProfileCard } from './ENSCreateProfileCard';
+export { GasCard } from './GasCard';
+export { LearnCard } from './LearnCard';
+export { ReceiveAssetsCard } from './ReceiveAssetsCard';
```

Comment:
I think we should avoid doing this. No need to have multiple ways to import a component, should just be `import { DPICard } from '@/components/cards/DPICard'`.

I think this pattern makes sense for a library author who only wants to expose certain exports, but doesn't make sense in an app like this.